The new Agent pre-release throws an error upon startup

laci@spark ~/apps $ ./UHK.Agent-1.2.6-linux-x86_64.AppImage
module.js:487
    throw err;
    ^

Error: Cannot find module '/tmp/.mount_UHK.Agtd2Qe8/resources/electron.asar/browser/init.js'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Function.Module.runMain (module.js:605:10)
    at startup (bootstrap_node.js:167:16)
    at bootstrap_node.js:589:3

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "decompress-tarbz2": "4.1.1",
     "devtron": "1.4.0",
     "electron": "1.8.4",
-    "electron-builder": "20.15.0",
+    "electron-builder": "20.26.0",
     "electron-debug": "1.5.0",
     "electron-devtools-installer": "2.2.3",
     "electron-log": "2.2.16",

